commit 86860896dc1841eb59cb95832d76a8093e8dc8c5
Author: Christopher Faulet <cfaulet@haproxy.com>
Date:   Tue Apr 30 10:55:38 2019 +0200

    MINOR: examples: Use right locale for the last changelog date in haproxy.spec
    
    The last changelog entry was stamped with the wrong locale.
    
    No need to backport, it is specific to 1.8

diff --git a/examples/haproxy.spec b/examples/haproxy.spec
index f3e0c7e0..fe5215d7 100644
--- a/examples/haproxy.spec
+++ b/examples/haproxy.spec
@@ -74,7 +74,7 @@ fi
 %attr(0755,root,root) %config %{_sysconfdir}/rc.d/init.d/%{name}
 
 %changelog
-* jeu. avril 25 2019 Christopher Faulet <cfaulet@haproxy.com>
+* Thu Apr 25 2019 Christopher Faulet <cfaulet@haproxy.com>
 - updated to 1.8.20
 
 * Mon Feb 11 2019 Willy Tarreau <w@1wt.eu>
